### ISSUE TITLE
Failing tests

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#1410]
There are some failing tests, due to the slight change of **subplot -M** default value.

```
	235 - exmod/ex01.sh (Failed)
	253 - exmod/ex19.sh (Failed)
	518 - modern/box.sh (Failed)
	521 - modern/panels.sh (Failed)
	522 - modern/polarpanels.sh (Failed)
	525 - modern/somepanels.sh (Failed)
	851 - subplot/tictactoe.sh (Failed)
```

**Comment by PaulWessel** [comment]
Yes, do you have access to the new PS or shall I run the tests locally and update?

**Comment by seisman** [comment]
It's better to run the tests locally and manually check them again.

**Comment by PaulWessel** [comment]
Closed by #1411.
